dataactvalidator/filestreaming/fieldCleaner.py

```diff
@@ -136,6 +136,10 @@ def clean_row(cls, row, long_to_short_dict, fields):
             if value is not None:
                 # Remove extra whitespace
                 value = value.strip()
+                # If field is string and has triple quotes, remove. Single quote because csv converts triple to single
+                if value.startswith('"') and value.endswith('"'):
+                    value = value[1:-1]
```

Comment:
could you just put `.strip()` at the end of this instead of making it a separate line? Or does that not work?

Reply:
Forgot to say it but comment is incorrect.